src/containers/explore/reducers/index.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * @flow
+ */
+
+import { Map, fromJS } from 'immutable';
+import { matchPath } from 'react-router';
+
+import exploreEntityDataReducer from './exploreEntityData';
+import exploreEntityNeighborsReducer from './exploreEntityNeighbors';
+
+import { RESET_REQUEST_STATES } from '../../../core/redux/actions';
+import {
+  ENTITY_NEIGHBORS_MAP,
+  RS_INITIAL_STATE,
+  SELECTED_ENTITY_DATA
+} from '../../../core/redux/constants';
+import { resetRequestStatesReducer } from '../../../core/redux/reducers';
+import { Routes, RoutingActions } from '../../../core/router';
+import {
+  EXPLORE_ENTITY_DATA,
+  EXPLORE_ENTITY_NEIGHBORS,
+  exploreEntityData,
+  exploreEntityNeighbors,
+} from '../actions';
+import type { RoutingAction } from '../../../core/router/actions';
+
+const { GO_TO_ROUTE } = RoutingActions;
+
+const INITIAL_STATE :Map = fromJS({
+  [EXPLORE_ENTITY_DATA]: RS_INITIAL_STATE,
+  [EXPLORE_ENTITY_NEIGHBORS]: RS_INITIAL_STATE,
+  [ENTITY_NEIGHBORS_MAP]: Map(),
+  [SELECTED_ENTITY_DATA]: undefined,
```

Comment:
I think this should probably be `Map()` instead of `undefined` ... feels like it would be "safer" 🤔 I should read through the rest of the changes.